src/locales/de/translation.json

```
@@ -482,6 +501,7 @@
     "inactiveAudit.table.emailDispatched": "E-Mail versendet",
     "agencyAdmins.title": "Agentur-Admins",
     "agencyAdmins.title.text": "{{userCount}} Agentur-Admins",
+    "tenantAdmins.title.text": "{{userCount}} Mandanten-Admins",
```

Biome (2.4.16)

[error] The key tenantAdmins.title.text was already declared.

(lint/suspicious/noDuplicateObjectKeys)